compel/plugins/include/uapi/std/log.h

```diff
@@ -26,5 +26,6 @@ extern void std_sprintf(char output[STD_LOG_SIMPLE_CHUNK], const char *format, .
 	__attribute__ ((__format__ (__printf__, 2, 3)));
 extern void print_on_level(unsigned int loglevel, const char *format, ...)
 	__attribute__ ((__format__ (__printf__, 2, 3)));
+extern int flog_encode_msg(int loglevel, unsigned int nargs, unsigned int mask, const char *format, ...);
```

Comment:
You should have commented out all logging from compel, why does flog forward declaration go here?

Reply:
There are log messages in criu too that had initially used compel’s print_on_level() to print the messages into the file. I converted all the pr_msg, etc to call flog_encode_msg() instead of print_on_level(). Now, these log messages search for the flog’s function in compel’s log.c file. Removing the function header gives me an error: External symbol in PIE?. This causes the build to fail. I am not able to figure out why it is searching in compel/plugin/std/log.c instead of the flog.c file.